GVFS/GVFS.Common/LibGit2RepoPool.cs

```diff
@@ -9,24 +9,38 @@ namespace GVFS.Common
     public class LibGit2RepoPool
     {
         private const int TryAddTimeoutMilliseconds = 10;
-        private const int TryTakeTimeoutMilliseconds = Timeout.Infinite;
+        private const int TryTakeWaitForever = Timeout.Infinite;
+        private const int TryTakeWaitShort = 10;

```

Comment:
The other names include the unit of measure, should this one keep that? `TryTakeWaitShortMilliseconds`?

Reply:
Actually, we should make the short wait be _no wait_ with 0 milliseconds, which means the units don't matter.